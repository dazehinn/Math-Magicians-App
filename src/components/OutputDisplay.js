import PropTypes from 'prop-types';

const OutputDisplay = ({ contented }) => (
  <div className="outputDisplay">{contented}</div>
);

OutputDisplay.propTypes = {
  contented: PropTypes.string,
};

OutputDisplay.defaultProps = {
  contented: '0',
};

export default OutputDisplay;
