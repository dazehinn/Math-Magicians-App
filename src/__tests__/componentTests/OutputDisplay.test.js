import renderer from 'react-test-renderer';
import OutputDisplay from '../../components/OutputDisplay';

describe('Display Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<OutputDisplay calcObj={{}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});