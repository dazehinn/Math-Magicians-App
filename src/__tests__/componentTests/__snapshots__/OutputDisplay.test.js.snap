// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component Tree 1`] = `
<div
  className="outputDisplay"
>
  0
</div>
`;
